CHANGELOG.md

```diff
@@ -111,3 +133,18 @@ database {
 }
 ```
 
+## 0.20
+
+* The default per-upload bytes size for GCS is now the minumum 256K
+instead of 64M. There is also an undocumented config key
+`google.upload-buffer-bytes` that allows adjusting this internal value.
```

Comment:
It's mentioned in the release notes, so I guess "undocumented" means it's not in the README?  Should we correct that?